cmd/sink/main.go

```diff
@@ -132,7 +140,14 @@ func main() {
 	if err != nil {
 		logger.Fatalf("Could not connect to the database: %s\n", err)
 	}
-	sink := NewSink(db, logger)
+	filters, err := expr.NewFilters(os.Getenv(mountPathEnvVar))
+	if err != nil {
+		log.Printf(
+			"Not all filters could be created from the ConfigMap. Some archive and delete operations will not execute until the errors are resolved: %s\n",
```

Comment:
Shouldn't this be "logger" instead of "log"?

All the other logging messages start with a lower case word, why is "Not" capitalized here?

Reply:
I was going to ask the opposite. Why aren't we using normal english, and capitalizing the start of messages?

Reply:
yes, that should be `logger`. I fixed that. I also capitalized all of the log messages

Reply:
> I fixed that. I also capitalized all of the log messages

We should standardize on this if that's what we want to do.